cs-couchbase/src/main/java/io/cloudslang/content/couchbase/actions/buckets/GetBucketStatistics.java

```diff
@@ -0,0 +1,101 @@
+
+import static org.apache.http.client.methods.HttpGet.METHOD_NAME;
+
+/**
+ * Created by Mihai Tusa
+ * 3/26/2017.
+ */
+public class GetBucketStatistics {
+    /**
+     * Retrieve all bucket information for a cluster.
+     * http://docs.couchbase.com/admin/admin/REST/rest-buckets-summary.html
+     *
+     * @param endpoint      Endpoint to which request will be sent. A valid endpoint will be formatted as it shows in
+     *                      bellow example.
+     *                      Example: "http://somewhere.couchbase.com:8091"
+     * @param username      Username used in basic authentication.
+     * @param password      Password associated with "username" input to be used in basic authentication.
+     * @param proxyHost     Optional - proxy server used to connect to Couchbase API. If empty no proxy will be used.
+     * @param proxyPort     Optional - proxy server port. You must either specify values for both proxyHost and proxyPort
+     *                      inputs or leave them both empty.
+     * @param proxyUsername Optional - proxy server user name.
+     * @param proxyPassword Optional - proxy server password associated with the proxyUsername input value.
+     * @param bucketName    Name of the bucket to retrieve details for
+     * @return A map with strings as keys and strings as values that contains: outcome of the action (or failure message
+     * and the exception if there is one), returnCode of the operation and the ID of the request
+     */
+    @Action(name = "Get Bucket Statistics",
+            outputs = {
+                    @Output(RETURN_CODE),
+                    @Output(RETURN_RESULT),
+                    @Output(EXCEPTION)
+            },
+            responses = {
+                    @Response(text = SUCCESS, field = RETURN_CODE, value = ReturnCodes.SUCCESS,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.RESOLVED),
+                    @Response(text = FAILURE, field = RETURN_CODE, value = ReturnCodes.FAILURE,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.ERROR, isOnFail = true)
+            })
+    public Map<String, String> execute(@Param(value = ENDPOINT, required = true) String endpoint,
+                                       @Param(value = USERNAME, required = true) String username,
+                                       @Param(value = PASSWORD, required = true, encrypted = true) String password,
+                                       @Param(value = PROXY_HOST) String proxyHost,
+                                       @Param(value = PROXY_PORT) String proxyPort,
+                                       @Param(value = PROXY_USERNAME) String proxyUsername,
+                                       @Param(value = PROXY_PASSWORD, encrypted = true) String proxyPassword,
+                                       @Param(value = BUCKET_NAME, encrypted = true) String bucketName) {
```

Comment:
you can add some inputs from security inputs from csHttpClient, like:
     ```
            @Param(HttpClientInputs.TRUST_ALL_ROOTS) String trustAllRoots,
            @Param(HttpClientInputs.X509_HOSTNAME_VERIFIER) String x509HostnameVerifier,
            @Param(HttpClientInputs.TRUST_KEYSTORE) String trustKeystore,
            @Param(HttpClientInputs.TRUST_PASSWORD) String trustPassword,
            @Param(HttpClientInputs.KEYSTORE) String keystore,
            @Param(HttpClientInputs.KEYSTORE_PASSWORD) String keystorePassword,
            @Param(HttpClientInputs.CONNECT_TIMEOUT) String connectTimeout,
            @Param(HttpClientInputs.SOCKET_TIMEOUT) String socketTimeout,
            @Param(HttpClientInputs.USE_COOKIES) String useCookies,
            @Param(HttpClientInputs.KEEP_ALIVE) String keepAlive
```
(these are some examples, you can add or subtract from them, whatever you think fit )